libs/documentation/src/lib/components/download/download.module.ts

```diff
@@ -11,9 +11,20 @@ import { DownloadComponent } from './demos/download/download.component';
 import { DownloadBasicModule } from './demos/download/download.module';
 import { FormlyDialog } from './demos/formlymodal/formly-modal.component';
 import { FormlyDialogModule } from './demos/formlymodal/formly-modal.module';
+import { FormlyCurvyDialog } from './demos/formlymodal-curvy/formlymodal-curvy.component';
+import { FormlyCurvyDialogModule } from './demos/formlymodal-curvy/formlymodal-curvy.module';
+
 
 declare var require: any;
 const DEMOS = {
+  formlymodalcurvy: {
+    title: 'Curvy Modal Dialog with Formly',
```

Comment:
Since this demo is actually demonstrating adding a field description with a link in the modal, it might make more sense to adjust the naming to reflect that.